ligo/token/fa2.mligo

```diff
@@ -0,0 +1,108 @@
+// SPDX-FileCopyrightText: 2021 Arthur Breitman
+// SPDX-License-Identifier: LicenseRef-MIT-Arthur-Breitman
+
+#include "../types.mligo"
+
+// -----------------------------------------------------------------
+// Helper
+// -----------------------------------------------------------------
+
+[@inline]
+let get_position_index (position_id, store : position_id * storage) : position_index =
+  match (Big_map.find_opt position_id store.position_indexes) with
+    | Some position_index -> position_index
+    | None -> ([%Michelson ({| { FAILWITH } |} : string * unit -> position_index)]
+          ("FA2_TOKEN_UNDEFINED", ()) : position_index)
+
+[@inline]
+let check_sender (from_ , store : address * storage): address =
+  if (Tezos.sender = from_) then from_
+  else
+    let key: operator = { owner = from_; operator = sender} in
```

Comment:
There's one more use of `Tezos.sender` here

Reply:
Fixed this now.